src/player.js

```diff
@@ -2558,15 +2559,15 @@ export default class Player extends FakeEventTarget {
   /**
    * Gets the track language that should be set by default.
    * @param {string} configuredLanguage - The configured language (can be also "auto").
-   * @param {?TextTrack} defaultTrack - The default track.
+   * @param {?(TextTrack|AudioTrack)} defaultTrack - The default track.
    * @private
    * @returns {string} - The track language to set by default.
    */
-  _getLanguage(configuredLanguage: string, defaultTrack: ?TextTrack): string {
+  _getLanguage(configuredLanguage: string, defaultTrack: ?(TextTrack | AudioTrack)): string {
```

Comment:
I think that writing it like in _setDefaultTrack, with a typed function and passing the tracks from outside, is cleaner than having an if inside the function.

And if we don't use language outside _setDefaultTrack anyway, you can also merge _setDefaultTrack with _getLanguage, or call _getLanguage from inside _setDefaultTrack.

Reply:
done